pkg/kv/kvserver/rangefeed/bench_test.go

```diff
@@ -41,19 +40,16 @@ const (
 // BenchmarkRangefeed benchmarks the processor and registrations, by submitting
 // a set of events and waiting until they are all emitted.
 func BenchmarkRangefeed(b *testing.B) {
-	for _, procType := range testTypes {
```

Comment:
the poor microbenchmark dashboard :-) I'm fine with this change but if you want continuity consider leaving the outer loop intact, even though it'll only loop over one element.

Reply:
Yeah, didn't know about that dashboard when I did this. Good catch.